src/api_manager/proto/server_config.proto

```diff
@@ -96,6 +96,12 @@ message ServiceControlConfig {
   // Timeout in milliseconds on service control allocate quota requests.
   // If the value is <= 0, default timeout is 5000 milliseconds.
   int32 quota_timeout_ms = 9;
+
+  // Config for log HTTP request headers.
```

Comment:
please add comment to say, this is comma separated multiple headers.  Maybe it is better to use repeated type

Reply:
Done.